files/ko/web/api/web_audio_api/advanced_techniques/index.html

```diff
@@ -12,91 +12,91 @@
 </div>
 
-<p>The interface consists of master controls, which allow us to play/stop the sequencer, and adjust the BPM (beats per minute) to speed up or slow down the "music".</p>
+<p>인터페이스는 시퀸서를 재생/정지할 수 있게 하는 주 제어기, 그리고 "음악"의 속도를 올리거나 내릴 수 있게 하는 BPM (beats per minute) 조절기로 이루어져 있습니다.</p>
 
-<p>There are four different sounds, or voices, which can be played. Each voice has four buttons, which represent four beats in one bar of music. When they are enabled the note will sound. When the instrument plays, it will move across this set of beats and loop the bar.</p>
+<p>재생될 수 있는 네 가지의 다른 음, 또는 소리가 있습니다. 각 소리는 네 개의 버튼을 가지고 있는데, 이는 음악의 한 마디에 있는 네 개의 비트를 나타냅니다. 이것들이 활성화되었을 때 노트는 소리를 낼 것입니다. 악기가 재생될 때, 이것은 이 비트들의 세트를 가로질러 이동하고 마디를 반복할 것입니다.</p>
 
-<p>Each voice also has local controls, which allow you to manipulate the effects or parameters particular to each technique we are using to create those voices. The techniques we are using are:</p>
+<p>각 소리는 또한 지역 제어기를 가지고 있는데, 이는 이펙트나 우리가 이 소리들을 만들기 위해 사용한 각 기법의 특정한 매개변수를 조작할 수 있게 해 줍니다. 우리가 사용하는 이 기법들은:</p>
 
 <table class="standard-table">
  <thead>
   <tr>
-   <th scope="col">Name of voice</th>
-   <th scope="col">Technique</th>
-   <th scope="col">Associated Web Audio API feature</th>
+   <th scope="col">소리의 이름</th>
+   <th scope="col">기법</th>
+   <th scope="col">연관된 Web Audio API 기능</th>
   </tr>
  </thead>
  <tbody>
   <tr>
-   <td>"Sweep"</td>
-   <td>Oscillator, periodic wave</td>
+   <td>"스윕"</td>
+   <td>오실레이터, 주기파</td>
    <td>{{domxref("OscillatorNode")}}, {{domxref("PeriodicWave")}}</td>
   </tr>
   <tr>
-   <td>"Pulse"</td>
-   <td>Multiple oscillators</td>
+   <td>"펄스"</td>
+   <td>다수의 오실레이터</td>
    <td>{{domxref("OscillatorNode")}}</td>
   </tr>
   <tr>
-   <td>"Noise"</td>
-   <td>Random noise buffer, Biquad filter</td>
+   <td>"노이즈"</td>
+   <td>무작위 노이즈 버퍼, 바이쿼드 필터</td>
    <td>{{domxref("AudioBuffer")}}, {{domxref("AudioBufferSourceNode")}}, {{domxref("BiquadFilterNode")}}</td>
   </tr>
   <tr>
-   <td>"Dial up"</td>
-   <td>Loading a sound sample to play</td>
+   <td>"다이얼 음"</td>
+   <td>재생할 사운드 샘플 로딩하기</td>
    <td>{{domxref("BaseAudioContext/decodeAudioData")}}, {{domxref("AudioBufferSourceNode")}}</td>
   </tr>
  </tbody>
 </table>
 
 <div class="note">
-<p><strong>Note</strong>: This instrument was not created to sound good, it was created to provide demonstration code and represents a <em>very</em> simplified version of such an instrument. The sounds are based on a dial-up modem. If you are unaware of how one sounds you can <a href="https://soundcloud.com/john-pemberton/modem-dialup">listen to one here</a>.</p>
+<p><strong>참고</strong>: 이 악기는 좋은 소리를 내기 위해 만들어지지 않았습니다. 이것은 데모 코드를 제공하기 위해 만들어졌고 이런 악기의 <em>아주</em> 단순화된 버전을 나타냅니다. 이 소리들은 다이얼 접속 모뎀에 기반합니다. 만약 여러분이 이것이 어떤 소리인지 모른다면 <a href="https://soundcloud.com/john-pemberton/modem-dialup">여기서 들어볼</a> 수 있습니다.</p>
```

Comment:
```suggestion
<p><strong>참고</strong>: 이 악기는 좋은 소리를 내기 위해 만들어지지 않았습니다. 이것은 데모 코드를 제공하기 위해 만들어졌고 이런 악기의 <em>아주</em> 단순화된 버전을 나타냅니다. 이 소리들은 다이얼 접속 모뎀에 기반합니다. 만약 여러분이 이것이 어떤 소리인지 모른다면 <a href="https://soundcloud.com/john-pemberton/modem-dialup">여기서 들어볼 수 있습니다.</a></p>
```
하이퍼링크의 위치를 조금 자연스럽게 해보았습니다:)